### PR TITLE
feat(dart_frog_cli): support lowercase r to reload

### LINKED ISSUE #893: feat(dart_frog_cli): support lowercase r to reload
**Description**

Currently, https://github.com/VeryGoodOpenSource/dart_frog/pull/814 only supports uppercase "R" to reload. Flutter also supports "r".

**Requirements**

- [ ] Allow "r" to reload

**Additional Context**

-https://github.com/VeryGoodOpenSource/dart_frog/blob/779e4fefb47677e95baa818aa071aec858ed3810/packages/dart_frog_cli/lib/src/commands/dev/dev.dart#L71

### PR DESCRIPTION
**Pull request by Yczar**
## Status

**READY*

## Description

 This change adds support lowercase r to reload.
 fixes #893 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by renancaraujo — APPROVED** [review]

File: packages/dart_frog_cli/lib/src/commands/dev/dev.dart [hunk 1]
```diff
@@ -68,7 +68,9 @@ class DevCommand extends DartFrogCommand {
 
     _stdinSubscription = stdin.listen(
       (event) {
-        if (event.length == 1 && event.first == 'R'.codeUnitAt(0)) {
+        if (event.length == 1 &&
+            (event.first == 'R'.codeUnitAt(0) ||
+                event.first == 'r'.codeUnitAt(0))) {
```
Comment (renancaraujo) [inline packages/dart_frog_cli/lib/src/commands/dev/dev.dart:73]: Actually we will need tests for this

**Review by renancaraujo — CHANGES_REQUESTED** [review]
Need tests

File: packages/dart_frog_cli/lib/src/commands/dev/dev.dart (hunk 1 above)
Comment (renancaraujo) [reply packages/dart_frog_cli/lib/src/commands/dev/dev.dart:73]: @Yczar would you need assitance to add tests for this?

**Comment by Yczar** [comment]
@renancaraujo done!

**Comment by Yczar** [comment]
Hey @renancaraujo were you able to check this out?

**Review by renancaraujo — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**